internal/storage/storage_handle.go

```diff
@@ -43,6 +43,17 @@ type StorageClientConfig struct {
 	HttpClientTimeout   time.Duration
 	MaxRetryDuration    time.Duration
 	RetryMultiplier     float64
+	UserAgent           string
+}
+
+type userAgentRoundTripper struct {
+	inner     http.RoundTripper
```

Comment:
we use the terminology wrapped for these scenarios

Reply:
fixed